errtracker/errtracker.go

```diff
@@ -64,10 +70,19 @@ func Report(snap, errMsg, dupSig string, extra map[string]string) (string, error
 		return "", nil
 	}
 
-	machineID, err := ioutil.ReadFile(machineID)
-	if err != nil {
-		return "", err
+	var machineID []byte
+	var err error
+	for _, id := range machineIDs {
+		machineID, err = ioutil.ReadFile(id)
+		if err == nil {
+			break
+		}
 	}
+
+	if len(machineID) == 0 {
+		return "", fmt.Errorf("Can not report as no machine id found")
```

Comment:
We always use "cannot" in error messages.

`cannot report error: no suitable machine-id file was found`